### ISSUE TITLE
Add more proposals

### ISSUE THREAD

**Issue by prondubuisi** [prondubuisi/accepted-gsoc-proposals#13]
Here is a link containing more proposals https://blog.sdslabs.co/gsoc/, confirm that the proposal link its still available, then add the proposal following the contribution guide line. I have already added proposals from the link up to 2017, so you can start from 2016

**Comment by Medlineme4** [comment]
Thanks for helping out

**Comment by prondubuisi** [comment]
> Thanks for helping out

Welcome. Don't forget to star the project!

**Comment by Medlineme4** [comment]
Ok. How do I do that?

Sent from Yahoo Mail on Android 
 
  On Sun, Mar 15, 2020 at 11:31 AM, Onyemenam Ndubuisi<notifications@github.com> wrote:   



Thanks for helping out


Welcome. Don't forget to star the project!

**Comment by prondubuisi** [comment]
> Ok. How do I do that? Sent from Yahoo Mail on Android On Sun, Mar 15, 2020 at 11:31 AM, Onyemenam Ndubuisi<notifications@github.com> wrote: Thanks for helping out Welcome. Don't forget to star the project! — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.

![tempFileForShare_20200315-114639](https://user-images.githubusercontent.com/22311928/76699967-d57a5f00-66b2-11ea-9d7f-47296a48bf28.jpg)

Click on star at the top of the project

**Comment by Medlineme4** [comment]
Alright
Sent from Yahoo Mail on Android 
 
  On Sun, Mar 15, 2020 at 11:48 AM, Onyemenam Ndubuisi<notifications@github.com> wrote:   



Ok. How do I do that? Sent from Yahoo Mail on Android On Sun, Mar 15, 2020 at 11:31 AM, Onyemenam Ndubuisinotifications@github.com wrote: Thanks for helping out Welcome. Don't forget to star the project! — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.




Click on star at the top of the project

**Comment by mubarak23** [comment]
I will work on checking the proposal and submit the working link for 2015 and 2014

**Comment by prondubuisi** [comment]
> I will work on checking the proposal and submit the working link for 2015 and 2014

Awesome, Assigning this to you, please feel free to make one commit at a time if that will be easier. Thanks once more! @mubarak23

**Comment by mubarak23** [comment]
if I understand this issue correctly, I will create a folder for each organization and create a .md file according to year the proposal was accepted

**Comment by prondubuisi** [comment]
> if I understand this issue correctly, I will create a folder for each organization and create a .md file according to year the proposal was accepted

Exactly! Then add Proposal link as well as Project link using the table structure already available.
